corehq/ex-submodules/casexml/apps/case/cleanup.py

```diff
@@ -62,7 +59,6 @@ def rebuild_case_from_forms(domain, case_id, detail):
     return FormProcessorInterface(domain).hard_rebuild_case(case_id, detail)
 
 
-@unit_testing_only
```

Comment:
i did think this was good idea. maybe we could add `i_know_what_im_doing=True` as an option to override it in non-testing code